Scrumdinger/Views/DetailEditView.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  DetailEditView.swift
+//  Scrumdinger
+//
+//  Created by Life Boss on 7/16/23.
+//
+
+import SwiftUI
+
+struct DetailEditView: View {
+    @State private var scrum = DailyScrum.emptyScrum
+    
+    var body: some View {
+        Form {
+            Section (header: Text("Meeting Info")) {
+                TextField("Title", text: $scrum.title)
+                HStack {
+                    Slider(value: $scrum.lengthInMinutesAsDouble, in: 5...30, step: 1) {
+                        Text("Length")
+                    }
+                    Spacer()
+                    Text("\(scrum.lengthInMinutes) minutes")
+                }
```

Comment:
☝️ Note the distinction of using `$scrum.` vs `scrum.`

Appending an initial `$` indicates that you are **binding** the property value, which is required in situations where you need read/write access to the state property:

https://github.com/meggra7/scrumdinger/blob/32223ef741e477a5e7bfe02df2463b0c69045139/Scrumdinger/Views/DetailEditView.swift#L18

You can and should omit the initial `$` when you need read-only access to the state property:

https://github.com/meggra7/scrumdinger/blob/32223ef741e477a5e7bfe02df2463b0c69045139/Scrumdinger/Views/DetailEditView.swift#L22